<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPlayerCard</class>
 <widget class="QDialog" name="DialogPlayerCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>选择牌</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="labelAvatar">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>11</y>
     <width>180</width>
     <height>152</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../fairsgsruntime/image/generals/avatar/7.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonHands">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>手牌</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonWuQi">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>武器</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonFangJu">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>防具</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonJiaYi">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>+1马</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonJianYi">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>-1马</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBaoWu">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>宝物</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonShanDian">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>闪电</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLeBuSiShu">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>乐不思蜀</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBingLiangCunDuan">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>兵粮寸断</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择一张牌。</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
